docs/NEW_LANGUAGE_FEATURE_INTEGRATION_PROCESS.md

```diff
@@ -36,28 +40,29 @@ on existing checks.
 
 **Examples**:
 
-- When the `RECORD_DEF` token was introduced, it was reasonable to look at all checks
-  that had `CLASS_DEF` in their acceptable tokens.
+- **Check updates**: When the `RECORD_DEF` token was introduced,
+ it was reasonable to look at all checks that had `CLASS_DEF` in their acceptable tokens.
 
-- When the `TEXT_BLOCK_CONTENT` token was introduced, it was reasonable to look at all checks
-  that had `STRING_LITERAL` in their acceptable tokens.
+- **Check updates**: When the `TEXT_BLOCK_CONTENT` token was introduced, 
+  it was reasonable to look at all checks that had `STRING_LITERAL` in their acceptable tokens.
 
-- When switch expressions were introduced, it was reasonable to look at all checks
-  that deals with expressions because `LITERAL_SWITCH` may now fall under `EXPR` token.
+- **Check updates**: When switch expressions were introduced, it was reasonable to look at all 
+  checks that deals with expressions because `LITERAL_SWITCH` may now fall under `EXPR` token.
   This requires analyzing the impact of this new child token on existing checks,
   such as `InnerAssignmentCheck`
 
-### Frequently Impacted Checks
+## Frequently Impacted Checks
 
 Certain checks are most likely to be impacted by new language features.
 We should analyze the impact on the following frequently affected checks:
 
+- [IllegalIdentifierName](https://checkstyle.org/checks/naming/illegalidentifiername.html#IllegalIdentifierName)
```

Comment:
Since this is now an abandoned check, I don't see much value in adding it now.

Reply:
done